share/harvest/base.py

```diff
@@ -115,7 +114,7 @@ def fetch_date(self, date: datetime.date, **kwargs):
         """
         return self.fetch_date_range(date - datetime.timedelta(days=1), date, **kwargs)
 
-    def fetch_date_range(self, start, end, limit=None, **kwargs):
+    def fetch_date_range(self, start, end, limit=None):
```

Comment:
The kwargs are there to allow passing arbitrary keyword arguments to `_do_fetch` without having to override any other functions. 
If additional kwargs are passed in, they should eventually get rejected by `_do_fetch`.
Is there any reason to remove them?